### PR TITLE
WEB: Update Python for Data Analysis book

### PR DESCRIPTION
**Pull request by mroeschke**

### CODE REVIEW

**Review by datapythonista — COMMENTED** [review]
Looks good. I don't remember exactly, but I think I comverted the image to a gif with custom palette to make it as small as possible last time. I see the new image is 30kb vs the 5kb of the old. Maybe not a big deal, bit we could try to do the same and keep the size of the pandas repo a bit smaller.

**Comment by mroeschke** [comment]
Sounds reasonable to convert the jpg to gif to reduce size. Not too familiar how to do that so any suggestions?

**Comment by datapythonista** [comment]
This is the original 30 kb file:

pydata_book

I'd use this version, it's 12kb:

pydata_book_16colors

 I can reduce it more, but the new cover has more colors and it's more sensitive than the previous to compression. While probably no big deal, the quality of a smaller version starts to be clearly worse. This is a 8kb image:

pydata_book_8colors

File: web/pandas/getting_started.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ and more about JupyterLab in the
...
 ## Books
 
-The book we recommend to learn pandas is [Python for Data Analysis](https://amzn.to/2KI5JJw),
+The book we recommend to learn pandas is [Python for Data Analysis](https://amzn.to/3DyLaJc),
 by [Wes McKinney](https://wesmckinney.com/), creator of pandas.
 
-<a href="https://amzn.to/2KI5JJw">
-    <img alt="Python for Data Analysis" src="{{ base_url }}/static/img/pydata_book.gif"/>
+<a href="https://amzn.to/3DyLaJc">
+    <img alt="Python for Data Analysis" src="{{ base_url }}/static/img/pydata_book.jpg"/>
```
Comment (datapythonista) [inline web/pandas/getting_started.md:58]: You need to revert this back to gif

File: web/pandas/index.html [hunk 2]
```diff
@@ -87,8 +87,8 @@ <h4>Follow us</h4>
                 </div>
 				<h4>Get the book</h4>
 				<p class="book">
-					<a href="https://amzn.to/2KI5JJw">
-						<img class="img-fluid" alt="Python for Data Analysis" src="{{ base_url }}/static/img/pydata_book.gif"/>
+					<a href="https://amzn.to/3DyLaJc">
+						<img class="img-fluid" alt="Python for Data Analysis" src="{{ base_url }}/static/img/pydata_book.jpg"/>
```
Comment (datapythonista) [inline web/pandas/index.html:91]: Same

**Comment by mroeschke** [comment]
Thanks, any thoughts whether we should include this for 1.5?

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @mroeschke 

This only affects the website, so I'd merge when ready, independently from 1.5 or any release, which as far as I know are unaffected by the website.

**Pull request merged**